bindings/c/test/mako/mako.cpp

```diff
@@ -544,7 +546,8 @@ int runOneTransaction(Transaction& tx,
 	if (needs_commit || args.commit_get) {
 		auto watch_commit = Stopwatch(StartAtCtor{});
 		auto f = tx.commit();
-		const auto rc = waitAndHandleError(tx, f, "COMMIT_AT_TX_END", args.transaction_timeout > 0);
+		const auto rc = waitAndHandleError(
+		    tx, f, "COMMIT_AT_TX_END", args.transaction_timeout_tx > 0 || args.transaction_timeout_db > 0);
```

Comment:
Introduce a boolean variable something like args.transaction_set and use it instead of that complex condition here and in other places.